pom.xml

```diff
@@ -68,7 +68,7 @@
     <!-- Package name to use when relocating shaded classes -->
     <gluten.shade.packageName>io.glutenproject.shaded</gluten.shade.packageName>
 
-    <substrait.version>0.5.0</substrait.version>
+    <substrait.version>0.22.0</substrait.version>
```

Comment:
0.22.0 seems to be one old tag, curious why use this one?

Reply:
Ah I didn't realize there was another release. I will update.